model/src/main/kotlin/Package.kt

```diff
@@ -58,6 +58,14 @@ data class Package(
      */
     val declaredLicenses: SortedSet<String>,
 
+    /**
+     * The list of contributors declared for this package.
+     *
+     * TODO: Remove default value after upgrading all package manager implementations.
+     */
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    val declaredContributors: SortedSet<String>? = null,
```

Comment:
We try to avoid nullable properties in the model, in this case an empty set is just as good as null and makes the property easier to use.
Instead of adding a default value here, you could already pass an empty set in all package managers and add the TODOs there, this would make it easier to track for which package managers this still needs to be implemented.
Would it make sense to capture this for projects as well? At least for some package managers like Maven we get the same metadata for projects as for packages. It might not be needed for copyright curations, but it would be more consistent.

Reply:
I added the `declaredContributors` field also in the `Project` class and my intention is to track also the contributors for the projects as well,  even if this is not required to solve the issue #3400.